crategen/models.py

```diff
@@ -1,5 +1,5 @@
-from pydantic import BaseModel, Field, validator,root_validator
-from typing import List, Optional, Dict, Any
+from pydantic import BaseModel, Field, validator, root_validator
+from typing import List, Optional, Dict
```

Comment:
When using Python 3.9 or newer (which I hope you do), please use `list` and `dict` instead of the uppercase variants `List` and `Dict` for type hints and Pydantic models.

Reply:
See https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
I will update the type hints to use list and dict instead of List and Dict to align with Python 3.9+ standards.